docs/content/installation/install_hostdefiner.md

```diff
@@ -0,0 +1,16 @@
+# Installing the host definer
+
+Install the HostDefiner custom resource to enable dynamic host definitions of the CSI driver nodes.
+
+For information on configuring the HostDefiner custom resource, see [Configuring the host definer](../configuration/configuring_hostdefiner).
+
+For more information on using dynamic host connectivity definition, see [Using dynamic host definition](../using/using_hostdefinition.md).
+
+The host definer can be downloaded and installed from the OpenShift web console and OperatorHub.io, as part of the operator and driver installation process. For more information, see [Installing the driver with the OpenShift web console](install_driver_openshift_web.md) and [Installing the driver with OperatorHub.io](install_driver_operatorhub.md).
+
+The host definer can also be installed directly with GitHub (see [Installing the host definer with GitHub](install_hostdefiner_github.md)).
+
+**Note:** 
+    
+- The host definer can be installed at any time.
+- Installing the host definer with GitHub requires all later host definer updates to be conducted manually.
```

Comment:
this is also true for the driver, so I'm not sure we should mention this explicitly

Reply:
@matancarmeli7  
said that I should add it in. I can either add in a similar note for the driver or remove both.
In this case I'm not sure the usage knowledge level of the customer and what is best

Reply:
ok, we can remove it

Reply:
removed